docs/contributions/developers/1_local_setup.md

```diff
@@ -156,7 +156,7 @@ This will allow you to:
 
 If you do run into issues, first perform a web search for your issue and see if it has already been answered. If you cannot find a solution, please search our [GitHub issues](https://github.com/Equal-Vote/bettervoting/issues) page for a solution and report the issue if it has not yet been reported.
 
-And as always, feel free to ask questions on the [#_software-dev slack channel](https://starvoting.slack.com/archives/C01EBAT283H).
+And as always, feel free to ask questions [here](https://slack.com/app_redirect?channel=C01EBAT283H).
```

Comment:
The new link doesn't seem to work for me. 

I think the old link still works, we just need to update the hyperlink text to refer to the _bettervoting slack channel.

Does the old link work for you?

Reply:
The new link works for me, but it puts up an intermediate page.  The old link works now for me -- and it goes directly to the _bettervoting slack channel.  So the old link is better.  I will fix it.l